### DISCUSSION TITLE
add global CSS prop wordWrap="break-word" by default

### DISCUSSION THREAD

**Discussion by tony9527167** [mantinedev/mantine#3915] [Feature requests]
We have to set every text-type components CSS wordWrap="break-word", it sucks,
please add global CSS prop wordWrap="break-word" by default, it is good for Responsive Design.

**Comment by dan0102dan** [comment]
Just create styles.css and add the following CSS rules:
```
span, p {
    word-break: break-word;
}
```

Import this stylesheet in your root component (e.g., App.js or index.js).
`import './styles.css';`
